RavenPy/recipe/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "0.3.0" %}
+
+package:
+  name: ravenpy
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/CSHS-CWRA/RavenPy.git
+  git_rev: v0.3.0
+
+build:
+  # noarch: python
+  skip: true  # [win]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    # I don't know if :: syntax does anything here. Can only be specified in conda-build command.
+    - zeitsperre::raven
+    - zeitsperre::ostrich
```

Comment:
Interesting.. what the interaction with the `-c` option on the command-line?

Reply:
From what I've read, this syntax will only work on `build` stage. I left it in here since it's a good reminder that there are packages named `raven` on other channels. For building RavenPy I use the following:
```
conda-build recipe/ --output-folder _build/ -c zeitsperre -c conda-forge --override-channels
```

Reply:
To be totally clear, I don't think specifying the channel in run does anything. It's completely ignored by `conda-build`.